client/lib/abtest/index.js

```diff
@@ -96,7 +104,20 @@ ABTest.prototype.init = function( name ) {
 	this.defaultVariation = testConfig.defaultVariation;
 	this.variationNames = variationNames;
 	this.experimentId = name + '_' + variationDatestamp;
-	this.allowAnyLocale = testConfig.allowAnyLocale === true;
+
+	this.localeTarget = 'en';
+	this.localeTargetExclude = false;
+
+	if ( testConfig.localeTargets ) {
+		if ( testConfig.localeTargets === 'any' ) {
+			this.localeTarget = false;
+		} else if ( testConfig.localeTargets === 'not-en' ) {
+			this.localeTargetExclude = true;
```

Comment:
So, if I'm following that `if`, the sole purpose of localeTargetExclude to exclude `en` right?  If yes, can we call it `excludeEnFromLocaleTarget` or`localeTargetExcludeEn` or something?